2_process.yml

```diff
@@ -91,3 +93,10 @@ targets:
       coarse_points_ind='2_process/out/storm_points_coarse.rds.ind')
   2_process/out/storm_points_interp.rds:
     command: gd_get('2_process/out/storm_points_interp.rds.ind')
+
+  2_process/out/normalize_streamdata.rds.ind:
+    command: normalize_streamdata(
+      ind_file = target_name,
+      raw_ind_file = '1_fetch/out/streamdata.rds.ind')
+  2_process/out/normalize_streamdata.rds:
```

Comment:
This is nitpicky, but could we name these output files `normalized_streamdata` with a `d`? To communicate that they're data products rather than functions?

Reply:
ah yup, good call